Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -1439,7 +1443,10 @@ public class LottieAnimationLayer: CALayer {
     let layerAnimation = CABasicAnimation(keyPath: "currentFrame")
     layerAnimation.fromValue = playFrom
     layerAnimation.toValue = playTo
-    layerAnimation.speed = Float(animationSpeed)
```

Comment:
Why would we remove `layerAnimation.speed = Float(animationSpeed)`? That sounds like it will break something.